pkg/config/env.go

```diff
@@ -189,6 +189,13 @@ var Config = struct {
 	HeaderAuthEnabled   bool   `env:"FLAGR_HEADER_AUTH_ENABLED" envDefault:"false"`
 	HeaderAuthUserField string `env:"FLAGR_HEADER_AUTH_USER_FIELD" envDefault:"X-Email"`
 
+	// Authenticate with basic auth
+	BasicAuthEnabled              bool     `env:"FLAGR_BASIC_AUTH_ENABLED" envDefault:"false"`
+	BasicAuthUsername             string   `env:"FLAGR_BASIC_AUTH_USERNAME" envDefault:""`
+	BasicAuthPassword             string   `env:"FLAGR_BASIC_AUTH_PASSWORD" envDefault:""`
+	BasicAuthPrefixWhitelistPaths []string `env:"FLAGR_BASIC_AUTH_WHITELIST_PATHS" envDefault:"/api/v1/flags,/api/v1/evaluation,/api/v1/evaluation/batch" envSeparator:","`
```

Comment:
Got it, since basicauth isn't like JWT auth, redirecting to a login site makes no sense

A few comments:
- For the prefix whitelist paths, `/api/v1/evaluation` already covers `/api/v1/evaluation/batch`, so you can keep the first.
- Can you briefly document the basic auth behavior for the default setting of BasicAuthPrefixWhitelistPaths and BasicAuthExactWhitelistPaths? For example, the UI access will promote a username/password login, and api will be fine to access the flags (note that it doesn't prevent people from directly curling /api/v1/flags to update flags)